src/state/slices/preferencesSlice/preferencesSlice.ts

```diff
@@ -23,7 +24,8 @@ const initialState: Preferences = {
     CosmosPlugin: getConfig().REACT_APP_FEATURE_PLUGIN_COSMOS,
     GemRamp: getConfig().REACT_APP_FEATURE_GEM_RAMP,
     FoxyInvestor: getConfig().REACT_APP_FEATURE_FOXY_INVESTOR,
-    ReduxLogging: getConfig().REACT_APP_REDUX_LOGGING
+    ReduxLogging: getConfig().REACT_APP_REDUX_LOGGING,
+    KeepKeySettings: getConfig().REACT_APP_KEEP_KEY_SETTINGS
```

Comment:
```suggestion
    KeepKeySettings: getConfig().REACT_APP_FEATURE_KEEPKEY_SETTINGS
```

Reply:
Renamed in https://github.com/shapeshift/web/pull/1341/commits/4ff5e2e7b910ed1038a6e6b9e6aca478245fb2af.